File: src/Examples/PB.SpecFlowMaster.Examples.Mbunit3/Steps.cs
using MbUnit.Framework;

// ReSharper disable once CheckNamespace
namespace PB.SpecFlowMaster.Examples
{
    public class Steps
    {
        protected void AreEqual(object expected, object actual)
        {
            Assert.AreEqual(expected, actual);
        }
    }
}

File: src/Examples/PB.SpecFlowMaster.Examples.VB/Steps.vb
Imports NUnit.Framework

' ReSharper disable once CheckNamespace
Namespace PB.SpecFlowMaster.Examples
    Public Class Steps
        Protected Sub AreEqual(ByVal expected As Object, ByVal actual As Object)
            Assert.AreEqual(expected, actual)
        End Sub
    End Class
End Namespace
